src/ConnectedVMware/test/AzConnectedVMwareCluster.Tests.ps1

```diff
@@ -15,16 +15,16 @@ if(($null -eq $TestName) -or ($TestName -contains 'AzConnectedVMwareCluster'))
 }
 
 Describe 'AzConnectedVMwareCluster' {
-    It 'CreateExpanded' {
-        New-AzConnectedVMwareCluster -Name $env.clusterName -ResourceGroupName $env.resourceGroupName -Location $env.location -ExtendedLocationName $env.extendedLocationName -ExtendedLocationType $env.extendedLocationType -InventoryItemId "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/azcli-test-rg/providers/Microsoft.ConnectedVMwarevSphere/VCenters/azcli-test-vc3/InventoryItems/domain-c644614"
+    It 'CreateExpanded' -Skip {
+        New-AzConnectedVMwareCluster -Name $env.clusterName -ResourceGroupName $env.resourceGroupName -Location $env.location -ExtendedLocationName $env.extendedLocationName -ExtendedLocationType $env.extendedLocationType -InventoryItemId "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/azcli-test-rg/providers/Microsoft.ConnectedVMwarevSphere/VCenters/azcli-test-vc/InventoryItems/domain-c649660"
     }
 
     It 'Get' {
-        $cluster = Get-AzConnectedVMwareCluster -ResourceGroupName $env.ResourceGroupName -Name $env.clusterName
-        $cluster.Name | Should -Be $env.clusterName
+        $cluster = Get-AzConnectedVMwareCluster -ResourceGroupName $env.ResourceGroupName -Name "Cluster1"
+        $cluster.Name | Should -Be "Cluster1"
```

Comment:
>"Cluster1" [](http://example.com/codeflow?start=35&length=10)

why do we need to hardcode these in tests? Can't these be used via env variables as before?

Reply:
removed the hardcoded value. Using variable.